docs/source/index.md

```diff
@@ -46,41 +56,71 @@ JupyterHub also provides a {doc}`REST API <reference/rest-api>`.
 The JupyterHub team and Project Jupyter value our community, and JupyterHub
 follows the Jupyter [Community Guides](https://jupyter.readthedocs.io/en/latest/community/content-community.html).
 
-## Contents
+---
+
+## Documentation Contents
```

Comment:
```suggestion
## Documentation Structure
```

I think this might be a more descriptive heading

Reply:
Thanks @sgibson91. I have made the edit.